noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -222,14 +222,14 @@ global BASE_OR_MERGE_PUBLIC_INPUTS_LENGTH = CONSTANT_ROLLUP_DATA_LENGTH + PARTIA
 
 global ENQUEUE_PUBLIC_FUNCTION_CALL_RETURN_LENGTH: u64 = 3 + CALL_CONTEXT_LENGTH;
 global GET_NOTES_ORACLE_RETURN_LENGTH: u64 = 674;
-global NOTE_HASHES_NUM_BYTES_PER_BASE_ROLLUP: Field = 2048;
-global NULLIFIERS_NUM_BYTES_PER_BASE_ROLLUP: Field = 2048;
-global PUBLIC_DATA_WRITES_NUM_BYTES_PER_BASE_ROLLUP: Field = 2048;
-global CONTRACTS_NUM_BYTES_PER_BASE_ROLLUP: Field = 32;
-global CONTRACT_DATA_NUM_BYTES_PER_BASE_ROLLUP: Field = 64;
-global CONTRACT_DATA_NUM_BYTES_PER_BASE_ROLLUP_UNPADDED: Field = 52;
-global L2_TO_L1_MSGS_NUM_BYTES_PER_BASE_ROLLUP: Field = 64;
-global LOGS_HASHES_NUM_BYTES_PER_BASE_ROLLUP: Field = 64;
+global NOTE_HASHES_NUM_BYTES_PER_BASE_ROLLUP: u64 = 32 * MAX_NEW_NOTE_HASHES_PER_TX;
+global NULLIFIERS_NUM_BYTES_PER_BASE_ROLLUP: u64 = 32 * MAX_NEW_NULLIFIERS_PER_TX;
+global PUBLIC_DATA_WRITES_NUM_BYTES_PER_BASE_ROLLUP: u64 = 64 * MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX; // 1 write is 64 bytes
```

Comment:
We could use constant propagation here ^ so I did that as we don't need the values to be literal in TS so I did that. We can't use constant propagation in other places because of [this](https://aztecprotocol.slack.com/archives/C03P17YHVK8/p1717167850738629).